src/__experimental__/components/form-field/index.js

```diff
@@ -1,3 +1 @@
-import FormField from './form-field.component';
-
-export default FormField;
+export { default } from './form-field.component';
```

Comment:
This is blowing my mind a little bit, didn't think to write a default `export` that way... But that's by the by.

Are these single line files necessary? Would it not make more sense to export directly from the component file?

Reply:
It is to aid imports being less verbose (`import 'form-field'` vs `form-field/form-field`) and in aid of IDEs searching for files and displaying filenames in tabs etc, rather than everything being in an index file.

Reply:
👍 No problem. I can see it's use for when you have lots of exports. It was a super-opinionated comment.